.prettierignore

```diff
@@ -1,2 +1,15 @@
+out/
+dist/
+node_modules/
+npm-debug.log
+yarn-error.log
+app/node_modules/
+.DS_Store
+.awcache
+.idea/
+.eslintcache
+
+app/static/common
+app/test/fixtures
+gemoji
 *.json
-*.md
```

Comment:
I'm pretty sure we don't want prettier messing with our md files.

Reply:
Why not?

Reply:
Because it adds unneeded friction. Our build shouldn't fail because something unrelated to building a project fails. An example is me adding something to our technical docs causing CI to fail.

Reply:
Coding style is unrelated to building the project, but we still enforce it 😄

This PR is just expanding our “style guide” (provided by Prettier) to the Markdown and SCSS files.

Reply:
So my question is if my MD style doesn't agree with Prettier, will CI fail and prevent me from merging? If the answer to that is yes, then this needs to be removed. We use these files internally and haven't had any issues with a lack of prettier's styling.

To state another way, we have had no issues with styling markdown.

Reply:
I'm in agreement with @iAmWillShepherd for a couple of reasons:

 - the output artefacts that are mission-critical are the code and stylesheets. Documentation should be easy to write, without requiring the same strictness
 - if all that Prettier is doing is enforcing a consistent line length, that shouldn't block CI
 - help ensure IDEs format Markdown files well when editing is more valuable

Reply:
> if all that Prettier is doing is enforcing a consistent line length, that shouldn't block CI

It also removes unnecessary linebreaks and formats code blocks and enforces consistent styles for lists, emphasis, headers, and several other things.

If you feel that requiring a consistent line length isn’t helpful, there’s [an option](https://prettier.io/docs/en/options.html#prose-wrap) to disable wrapping.

Reply:
@j-f1 the line length thing is just the one I picked out.

I'm fine with using prettier to format Markdown when editing, but enforcing it on CI adds a barrier to contribution that I feel is unnecessary.

Reply:
How about adding a new npm script to prettify the Markdown, but remove the check from the CI build (or list the non-pretty files, but not fail the build)?

Reply:
I think that works. @iAmWillShepherd?

Reply:
That's fine, but I don't find having a script for this to be useful. I don't think we've had any issues with our markdown files to date and I have a hard time caring about consistent styling for these files. I am, however, willing to defer this one so long as this won't block me from merging code that doesn't meet prettier's standards.